src/components/CellProduct.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+
+import { StyledText } from './StyledText'
+import { ProductIcon } from './icons/Icons'
+import { colors } from '../config/themes/appThemes'
+
+interface CellProductProps {
+  name: string
+  numberSales: number
+  price: number
+}
+
+export const CellProduct: React.FC<CellProductProps> = (props) => {
+  const { name, numberSales, price } = props
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.rowContainer}>
+        <View style={styles.icon}>
+          <ProductIcon width={90} height={90} color={colors.redBase} />
+        </View>
+        <StyledText style={styles.productName} body2 neutral100>
+          {name}
+        </StyledText>
+      </View>
+      <View style={styles.rowContainer}>
+        <StyledText style={styles.price} body2 neutral100>
+          $ {price}
+        </StyledText>
+        <StyledText style={styles.dot} body2 neutral100>
```

Comment:
No es necesario poner en un `style={styles.dot}`
```suggestion
        <StyledText body2 neutral100>
```